spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -1,8 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe InvenioRdmRecordConverter do
-  let(:user) { FactoryGirl.create(:user) }
-  let(:generic_file) { make_generic_file(user, doi: ["doi:123/ABC"], resource_type: ["Account Books"]) }
+  let(:user) { FactoryGirl.create(:user, username: "usr1234") }
+  let(:assistant) { FactoryGirl.create(:user, username: "ast9876") }
+
+  before do
+    ProxyDepositRights.create(grantor_id: assistant.id, grantee_id: user.id)
+  end
```

Comment:
Had difficulty setting this up, but the users needed to be created first, then the rights needed to be set up before making the GenericFile with `proxy_depositor` and `on_behalf_of`

Reply:
Thanks for the explanation. Glad you were able to find a solution :D